src/main/kotlin/exercise2/task1/FindPairOfHighestSum.kt

```diff
@@ -18,7 +18,60 @@ import org.jetbrains.exercise2.task3.findPairWithBiggestDifference
  */
 
 internal fun List<Int>.findHighestSumPair(): Pair<Int, Int> {
-    TODO("Implement me!!")
+
+
+//    for ( i in 0  ..this.size - 1){
+//        if(this[i] >= first){
+//            first = this[i]
+//        }
+//    }
+//
+//    for ( i in 0  .. this.size - 1){
+//        if(this[i] > second && this[i] < first){
+//            second = this[i]
+//        }
+//    }
+
+//    var first = this[0]
+//    var second = this[0]
+//    println("Velicina " + this.size)
+//    for (i in 0 ..< this.size -1){
+//        if (this[i+1] >= first ) first = this[i+1]
+//        if ((this[i] > second) && (this[i] < first)) second = this[i]
+//    }
+//    if ((this[lastIndex] > second) && (this[lastIndex] < first)) second = this[lastIndex]
+//    println("Prvi: " + first)
+//    println("Drugi: " + second)
+
+
+//    Prlazak kroz dve petlje
+//    var first = this[1]
+//    var second = this[0]
+//    for ( i in indices){
+//        if(this[i] >= first){
+//            first = this[i]
+//        }
+//    }
+//
+//    for ( i in indices){
+//        if((this[i] > second) && (this[i] < first)){
+//            second = this[i]
+//        }
+//    }
+
+//    Prolazak kroz jednu petlju
+    var first = this[0]
+    var second = this[0]
+    for (i in 0 ..< this.size -1){
+        if (this[i+1] >= first ){
+            second = first
+            first = this[i+1]
+        }
+        else if ((second < this[i+1]) && second < first){
+            second = this[i+1]
+        }
+    }
+    return Pair<Int, Int>(first, second)
```

Comment:
The `<Int, Int>` can be omitted since the compiler is smart enough to infer the generic type.

Also, there is an alternative way to declare a pair, and that is by using `to` infix extension function:
```first to second```